config/ct.config

```diff
@@ -1,6 +1,8 @@
 [
     {hpr, [
         {data_dir, "data"},
+        %% See https://github.com/helium/HIP/blob/main/0102-helium-educational-lns.md
+        {hpr_free_net_ids, ["C00053"]},
```

Comment:
If we put `16#C00053` in the settings, we can compare without needing to stringify.

Reply:
ok